### PR TITLE
Remove ../ from relative path when already in base path's root

### PR DESCRIPTION
**Pull request by SenseException**
This PR was created based on the comments of #15. When there is no way back in the directory structure of the basePath, because we are already in its root, then the relative path shouldn't contain `../` or `./` anymore.

### CODE REVIEW

File: tests/PathTest.php [hunk 1]
```diff
@@ -823,6 +823,39 @@ public function provideMakeRelativeTests()
             array('phar://C:/webmozart/css/style.css', 'phar://C:/webmozart/puli', '../css/style.css'),
             array('phar://C:/css/style.css', 'phar://C:/webmozart/puli', '../../css/style.css'),
 
+            // already relative + already in root basepath
+            array('../style.css', '/', 'style.css'),
+            array('./style.css', '/', 'style.css'),
+            array('../../style.css', '/', 'style.css'),
+            array('../style.css', 'C:\\', 'style.css'),
```
Comment (webmozart) [inline tests/PathTest.php]: I think the test cases with backslashes should also have backslashes on the left side.

**Comment by webmozart** [comment]
Awesome, thanks! :)

File: tests/PathTest.php (hunk 1 above)
Comment (SenseException) [inline tests/PathTest.php]: Backslashes are now on path's side too.

**Comment by webmozart** [comment]
Thanks! :)

**Pull request merged**